.github/pull-request-template.md

```diff
@@ -0,0 +1,17 @@
+Fixes #
+
+## Proposed Changes
```

Comment:
.github/pull-request-template.md:3: MD002 First header should be a top level header
.github/pull-request-template.md:5: MD003 Header style
.github/pull-request-template.md:15: MD014 Dollar signs used before commands without showing output
.github/pull-request-template.md:5: MD022 Headers should be surrounded by blank lines
.github/pull-request-template.md:9: MD036 Emphasis used instead of a header